nix/sources.json

```diff
@@ -15,12 +15,24 @@
         "branch": "fix-yarn2nix-for-nix-2.5-6_29769d2a1390d294469bcc6518f17931953545e1",
         "description": "Nix Packages collection",
         "homepage": "",
-        "owner": "rhinofi",
+        "owner": "deversifi",
```

Comment:
was the change back to `deversifi` intentional?

Reply:
No, not sure how it got there